engine/src/test/scala/cromwell/engine/WorkflowDescriptorSpec.scala

```diff
@@ -126,6 +127,7 @@ class WorkflowDescriptorSpec extends FlatSpec with Matchers with ScalaFutures {
       Map.empty,
       None)
 
+    implicit val patienceConfig = PatienceConfig(timeout = Span(10, Seconds), interval = Span(15, Millis))
```

Comment:
@Horneth perhaps try adding this to your branch if you push any more changes to see if this helps with your test failure too?